query-node/mappings/src/content/curatorGroup.ts

```diff
@@ -116,19 +120,49 @@ export async function content_CuratorRemoved({ store, event }: EventContext & St
     return inconsistentState('Non-existing curator group removal requested', curatorGroupId)
   }
 
-  const curatorIndex = curatorGroup.curators.findIndex((item) => item.id.toString() === curatorId.toString())
-
-  // ensure curator group exists
-  if (curatorIndex < 0) {
-    return inconsistentState('Non-associated curator removal from curator group requested', curatorId)
-  }
-
-  // update curator group
-  curatorGroup.curators.splice(curatorIndex, 1)
-
   // save curator group
   await store.save<CuratorGroup>(curatorGroup)
```

Comment:
Don't think we need to save `CuratorGroup` here?

Reply:
A relic from the previous implementation that's no longer needed. Removed.